sig-node/CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Contributing on Sig-Node
+
+Welcome!
+
+## For Kubernetes Contributions
+
+https://github.com/kubernetes/community/tree/master/contributors/guide#contributor-guide
+
+### For Enhancements 
+
+https://github.com/kubernetes/enhancements/tree/master/keps/sig-node
+
+#### Helpful Code Links for Sig-Node
+
+https://github.com/kubernetes/kubernetes/tree/master/cmd/kubelet
+https://github.com/kubernetes/kubernetes/tree/master/pkg/kubelet
+
+##### Here are some tasks created to help new contributors learn about the Kubelet
+
+    Task #1 : Compile kubelet
+        See tips in the root Makefile
+    Task #2 : Run a single unit test
+        See tips in the root Makefile
+    Task #3 : Explore update/verify scripts
+        hack/update-gofmt.sh + hack/verify-gofmt.sh
+        hack/update-bazel.sh + hack/verify-bazel.sh
+    Task #4 : Explore dependencies
```

Comment:
dependency management https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/vendor.md

Reply:
Yeah, this is helpful